src/ibex_bluesky_core/fitting.py

```diff
@@ -736,16 +746,14 @@ def guess(
                 x0 = x[-1]  # Picked as it can't be 0
 
             p = 1  # Expected value, not likely to change
-
-            const = 3  # largest difference in erfc function is between -1.5 and 1.5
-            w = np.abs(x[index_max_y] - x[index_min_y]) / const
+            w = 1 / calculate_erf_stretch(x, y, erfc=True, pre_sorted=True)
 
             init_guess = {
                 "b": lmfit.Parameter("b", b),
                 "r": lmfit.Parameter("r", r, min=0),
-                "x0": lmfit.Parameter("x0", x0, min=0),
+                "x0": lmfit.Parameter("x0", x0, min=0.1),
                 "p": lmfit.Parameter("p", p, min=0),
-                "w": lmfit.Parameter("w", w, min=0),
+                "w": lmfit.Parameter("w", w, min=0.1),
```

Comment:
These changes to `min` don't look right! were they just for testing?

Reply:
Unless you were having an actual problem with this, set to zero, or float_min if you _must_ I guess. But a fit with width or `x0` < 0.1 is entirely plausible.